ViAn/Filehandler/filehandler.cpp

```diff
@@ -98,54 +70,35 @@ QDir FileHandler::get_work_space(){
  * Reads filehandler from a json object.
  */
 void FileHandler::read(const QJsonObject &json){
-    QJsonArray json_projs = json["open"].toArray();
-    for(int i = 0; i != json_projs.size(); i++){
-        QJsonObject json_path = json_projs[i].toObject();
-        std::string path = json_path["path"].toString().toStdString();
-        open_project(load_project(path)->id);
-    }
+//    QJsonArray json_projs = json["open"].toArray();
+//    for(int i = 0; i != json_projs.size(); i++){
+//        QJsonObject json_path = json_projs[i].toObject();
+//        std::string path = json_path["path"].toString().toStdString();
+//        open_project(load_project(path)->id);
+//    }
 }
 /**
  * @brief FileHandler::write
  * @param json
  * Writes filehandler to a json object
  */
 void FileHandler::write(QJsonObject &json){
-    QJsonArray json_projs;
-    for (auto it = open_projects.begin();  it != open_projects.end(); it++) {
-        QJsonObject json_path;
-        ID id = *it;
-        Project* proj = get_project(id);
-        QDir dir = get_dir(proj->dir);
-        QString path;
-        if(this->save_format == BINARY ) path = dir.absoluteFilePath((proj->name + ".dat").c_str());
-        else path = dir.absoluteFilePath((proj->name + ".json").c_str());
-
-        json_path["path"] = path;
-        json_projs.append(json_path);
-    }
-    json["open"] = json_projs;
+//    QJsonArray json_projs;
+//    for (auto it = open_projects.begin();  it != open_projects.end(); it++) {
+//        QJsonObject json_path;
+//        ID id = *it;
+//        Project* proj = get_project(id);
+//        QDir dir = get_dir(proj->dir);
+//        QString path;
+//        if(this->save_format == BINARY ) path = dir.absoluteFilePath((proj->name + ".dat").c_str());
+//        else path = dir.absoluteFilePath((proj->name + ".json").c_str());
+
+//        json_path["path"] = path;
+//        json_projs.append(json_path);
+//    }
+//    json["open"] = json_projs;
 }
```

Comment:
These finctions should be removed (not in comments )if you dont need them.
Do you have any special resaon for keeping them?